docs/update-covid-local-restriction-data.md

```diff
@@ -0,0 +1,67 @@
+# Update COVID Local Restriction Data
+
+### Very brief description of how the checker works
+
+- The postcode checker makes calls to Mapit to get information about the submitted postcode.
+- The Mapit response contains `areas` that contain `GSS` codes for the postcode.
+- We select the GSS code for the lower tier local authority of the areas returned.
+- We then use the GSS code to query our local restriction YAML [data](https://github.com/alphagov/collections/blob/master/lib/local_restrictions/local-restrictions.yaml) to figure out which restriction the `GSS` code is under.
+
+### How to update the local restriction data
+
+[Here is an example PR!](https://github.com/alphagov/collections/pull/2009/files)
+
+In order to update the local restriction data, you will need to be given the `GSS` codes and new alert levels of the places for which restrictions are changing.
+
+You will also be told the time and date at which the alert levels will change.
+
+We display both the current and upcoming restrictions to a user IF we have upcoming restrictions for their postcode.
+
+![Image of results page with changing alert level](docs/alert-level-changing.png)
+
+Otherwise we just display the current restrictions. This is determined by the `start_date` and `start_time`.
+
+![Image of results page without changing alert level](docs/alert-level-no-upcoming-changes.png)
+
+Example of somewhere that just has current restrictions:
+
+```
+E08000012:
+  name: Liverpool City Council
+  restrictions:
+    - alert_level: 3
+      start_date: 2020-10-12
+      start_time: "00:01"
+```
+
+Example of somewhere with current and upcoming restrictions:
+
+```
+E08000010:
+  name: Wigan Borough Council
+  restrictions:
+    - alert_level: 2
+      start_date: 2020-10-12
+      start_time: "00:01"
+    - alert_level: 3
+      start_date: 2020-10-23
+      start_time: "00:01"
+```
+
+To update the restriction data, you will need to [add a list to the YAML](https://github.com/alphagov/collections/blob/master/lib/local_restrictions/local-restrictions.yaml) for the given GSS codes that contain:
+
+- the new `alert_level`.
+- the `start_date` for when the restrictions will be introduced.
+- the `start_time` for when the restrictions will be introduced.
```

Comment:
We should also mention, that a "new" entry should be added for the upcoming changes. Almost overwrote some existing ones not 5 minutes ago! 😱

Reply:
Thanks @leenagupte! have added something about adding rather than overwriting!